### DISCUSSION TITLE
Large MRF File Size

### DISCUSSION THREAD

**Discussion by mjackson2BH** [CMSgov/hospital-price-transparency#127] [Q&A] [answered]
Does anyone know if CMS will issue a grace period for the July 1 requirements? I have our files in the correct layout with additional elements however our developers are having to work on an increase in file size to upload to our website, in addition to allowing for zip file type uploads. Seems to be an issue with several health systems.

**Comment by RidenourK** [comment, marked as answer]
@mjackson2BH. Thank you for your question. 

As finalized in the CY2024 OPPS/ASC final rule, beginning July 1, 2024, all hospitals must adopt a CMS template and encode its standard charge information in it in the form and manner finalized in the [Hospital Price Transparency – Data Dictionary GitHub repository](https://github.com/CMSgov/hospital-price-transparency). CMS has begun enforcing these requirements as of July 1, 2024.
